src/routes/assemblies/[id]/+page.svelte

```diff
@@ -1,7 +1,15 @@
 <script lang="ts">
-	export let data;
+	import VoteCard from '$components/votes/VoteCard.svelte';
```

Comment:
Can you revert changes on this page because I want to throw all the data for another dev in the internal team 🙏